addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -168,7 +168,7 @@ export class HtmlField extends Component {
             this.dynamicPlaceholder?.setElementRef(this.wysiwyg);
         });
         onWillUnmount(() => {
-            if (!this.props.readonly && this._isDirty()) {
+            if (!this.props.readonly && !this.props.record.isNew && this._isDirty()) {
```

Comment:
Is this right though? What happens when you close the tab on a new record then? It's not saved since we did not commit the changes?

Reply:
@dmo-odoo  , I changed the fix in the source of the traceback which makes sense and does solve the problem , let me know what you think

Reply:
Way better, thanks! 👍